lecturer/junit_intro/src/test/java/by/jrr/start/CalculatorTest.java

```diff
@@ -33,55 +33,46 @@ public static void GlobalTearDown() {
     }
 
     @Test
-    public void sum() {
-        int a = 4;
-        int b = 3;
-
-        int expected = 7;
-        int actual = calculator.sum(a, b);
-
+    public void testDivision() {
+        int a = 12;
+        int b = 4;
+        int expected = 3;
+        int actual = calculator.division(a,b);
         assertEquals(expected, actual);
     }
 
     @Test
-    public void subs() {
-        int a = 4;
-        int b = 3;
-
-        int expected = 1;
-        int actual = calculator.subs(a, b);
-
+    public void testMultiplikation() {
+        int a = 3;
+        int b = 4;
+        int expected = 12;
+        int actual = calculator.multiplikation(a,b);
         assertEquals(expected, actual);
-
     }
 
     @Test
-    public void calculateSumThenSubs() {
-        int a = 4;
-        int b = 3;
-        int c = -5;
-
-        int expected = 12;
-        calculator.setResult(calculator.sum(a, b));
-        calculator.setResult(calculator.subs(calculator.getResult(), c));
-
+    public void calculateMultiplicationThenDivision() {
+        int a = 3;
+        int b = 4;
+        int c = 2;
+        int expected = 0;
         int actual = calculator.getResult();
-
         assertEquals(expected, actual);
+        calculator.setResult(calculator.multiplikation(a,b));
+        calculator.setResult(calculator.division(calculator.getResult(),c));
     }
 
     @Test
-    public void calculateSubsThenSum() {
-        int a = 4;
+    public void calculateDivisionThenMultiplikation() {
+        int a = 6;
         int b = 3;
-        int c = -5;
-
-        int expected = -4;
-        calculator.setResult(calculator.subs(a, b));
-        calculator.setResult(calculator.sum(calculator.getResult(), c));
-
+        int c = 2;
+        int expected = 0;
```

Comment:
defiantly expected not 0!

Reply:
I fixed it in my new pull request https://github.com/javaGuruBY/rpaS3/pull/195/files.